docs/Animation.md

```diff
@@ -1,7 +1,9 @@
-# Animation
+# Animation ![https://img.shields.io/badge/-Deprecated-red](https://img.shields.io/badge/-Deprecated-red)
 
 ## Description
 
+:warning: **The Product Price block has been deprecated**. Although support for this block is still granted.
+
```

Comment:
```suggestion
:warning: **The Animation block has been deprecated** (support is still granted).

```

Reply:
:warning: **The Animation block has been deprecated** (support is still given).